<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisDialog</class>
 <widget class="QDialog" name="AxisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>请选择生成隧道中轴线方法</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_projection">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>双向投影法</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_normal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>法向量法</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
